Probesys/bileto
# new: Add a message when opening a ticket

Related to https://github.com/Probesys/bileto/issues/42

Changes proposed in this pull request:

- Create the `Message` entity (with its repository + factory)
- Add a message input to the new ticket form
- Provide TinyMCE as WYSIWYG editor for messages
- List the messages on the ticket page

How to test the feature manually:

1. Start a new ticket, check that an editor is appearing
2. Write content with editor features (bold, lists, etc.)
3. Validate the form, check the message appears on the ticket page

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
